.github/workflows/main.yml

```diff
@@ -21,6 +21,12 @@ on:
 env:
   CI_CD_DART_SCRIPTS_PACKAGE_PATH: "tools/sz_repo_cli/"
 
+# Declare default permissions as none.
+#
+# Having workflows without setting the permission for the workflow is considered
+# a bad security practice and it is causing alerts from our scanning tools.
+permissions: {}
```

Comment:
```suggestion
# Set permissions to none.
#
# Using the broad default permissions is considered a bad security practice 
# and would cause alerts from our scanning tools.
permissions: {}
```